app/controllers/idv/doc_auth_controller.rb

```diff
@@ -68,13 +64,5 @@ def do_meta_refresh(meta_refresh_count)
     def flow_session
       user_session['idv/doc_auth']
     end
-
-    def check_for_outage
-      if VendorStatus.new.any_ial2_vendor_outage?
-        session[:vendor_outage_redirect] = current_step
-        session[:vendor_outage_redirect_from_idv] = true
-        redirect_to vendor_outage_url
```

Comment:
There are still a few stray references to the vendor outage page.

One reference is [in IdV](https://github.com/18F/identity-idp/blob/e990ee35a3ce6edb383a2b53ac18118fbe7a419a/app/controllers/idv/phone_controller.rb#L70), which @eric-gade and @solipet should be addressing in a separate PR

The other two are outside IdV and I think they need to stay, since the language on the new page ("$sp needs to make sure you are you") doesn't necessarily apply to those use cases.

- [`Users::PhoneController`](https://github.com/18F/identity-idp/blob/e990ee35a3ce6edb383a2b53ac18118fbe7a419a/app/controllers/users/phones_controller.rb#L33)
- [`Users::TwoFactorAuthenticationController`](https://gsa-tts.slack.com/archives/C5QUGUANN/p1678730283382089?thread_ts=1678728412.687009&cid=C5QUGUANN)

Reply:
Hm, I must have missed those references. Sorry for the noise!